charts/lws/templates/certmanager/certificatemetrics.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.enableCertManager }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  labels:
+    app.kubernetes.io/instance: serving-cert
+    {{- include "lws.labels" . | nindent 4 }}
+  name: lws-metrics-cert
+  namespace: {{ .Release.Namespace }}
+spec:
+  dnsNames:
+    - lws-controller-manager-metrics-service.{{ .Release.Namespace }}.svc
+    - lws-controller-manager-metrics-service.{{ .Release.Namespace }}.svc.cluster.local
```

Comment:
It would be better if the service name referenced a helm variable.

Reply:
I think, users prefer simply applying these configurations to work with whatever set as default. Probably there is no functional need to configure the service name. I'll defer the decision to approvers.

Reply:
I am not saying it has to be configurable. I just mean to share a variable so we do not have to harcdode the name in multiple places.